selfdrive/ui/qt/offroad/networking.cc

```diff
@@ -43,29 +44,39 @@ std::string exec(const char* cmd) {
 
 // Networking functions
 
-Networking::Networking(QWidget* parent, bool show_advanced) : QWidget(parent){
+Networking::Networking(QWidget* parent, bool show_advanced) : QWidget(parent), show_advanced(show_advanced){
+  s = new QStackedLayout;
+
+  QLabel* warning = new QLabel("Network manager is inactive!");
+  warning->setStyleSheet(R"(font-size: 65px;)");
+
+  s->addWidget(warning);
+  setLayout(s);
+
+  startup_timer = new QTimer(this);
```

Comment:
why do we need another timer? can't we just use the refresh timer?

Reply:
Do you think that would make the code better? I can do it, no problem.